docs/source/contributing/developing-core.md

```diff
@@ -126,9 +127,18 @@ corepack prepare pnpm@latest --activate
 or using `npm`:
 
 ```shell
-npm install -g pnpm
+npm install -g pnpm@latest
```

Comment:
Verifying that this worked for me on macOS 13.6.3, but the corepack command did not, when I attempted to verify the latest version.